keys/constants.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Peter Mattis (peter.mattis@gmail.com)
+
+package keys
+
+import "github.com/cockroachdb/cockroach/proto"
+
+// Constants for system-reserved keys in the KV map.
+var (
+	// KeyLocalPrefix is the prefix for keys which hold data local to a
```

Comment:
`KeyMax` is not movable without additional lifting because `proto/data.go` uses it in `proto.Key.PrefixEnd()`. Might be possible to move `proto.Key` to the `keys` package, but I'm not going to attempt that in this PR.